build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -171,55 +171,16 @@ steps:
       pushd ../azuredatastudio-darwin
```

Comment:
I feel like it'd be nice to move the other signing steps into the yaml you made as well so everything signing related is in that file

Reply:
So I was discussing this with Alan before - and this is just how I viewed it was that we're spreading them to different tasks based on the chance of failure rather than the "job" each task does.

My thought process was the codesign inside the normal build file doesn't depend on an external service, so no point in moving it around.

Reply:
also, i want to minimize the files that we upload, the manual signing should stay as it happens before the archive step.